src/transformStateWithClones.js

```diff
@@ -7,7 +7,30 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let cloneState = { ...state };
+  const clonestateArray = [];
```

Comment:
```suggestion
  const cloneStateArray = [];
```
don't forget to use camel-case